dcs/src/test/jdbc_test/src/test/java/org/trafodion/jdbc_test/TestBoolean.java

```diff
@@ -75,9 +75,17 @@ public void JDBCBoolean() throws SQLException {
             e.printStackTrace();
             assertNull(e.getMessage());
         } finally {
-            stmt.close();
-            prepStmt.close();
-            conn.close();
+            if (stmt != null) {
+                stmt.close();
+            }
+
+            if (prepStmt != null) {

```

Comment:
Could the stmt.close() throw an exception, preventing us from trying to close prepStmt and conn?